run_config_input.py

```diff
@@ -216,6 +220,17 @@ def run_normal_config_file():
         + str(config["predicted_value_option"])
     )
 
+    if run_in_parallel:
+        available_gpus = get_gpus_list()
+        model = torch.nn.parallel.DistributedDataParallel(
+            model, device_ids=["cuda:" + str(int(world_rank) % len(available_gpus))]
```

Comment:
This breaks on CPU. And in general, shouldn't we call `get_gpu` (`get_device`) here instead of doing this directly?

Reply:
Changed as suggested